### PR TITLE
ETS-566 - Add prefix

### PR DESCRIPTION
**Pull request by Engerim**
* use prefix for kubernetes namespace
* only shutdown prefixed namespaces

### CODE REVIEW

**Comment by dasrick** [comment]
👍

File: loader/config.go [hunk 1]
```diff
@@ -66,6 +70,7 @@ type Config struct {
 	Cleanup                  Cleanup
 	DNS                      DNSConfig `yaml:"dns"`
 	Database                 Database
+	Namespace				 Namespace
```
Comment (jazzzzy) [inline loader/config.go]: something seems broken here

File: service/app.go [hunk 2]
```diff
@@ -44,6 +45,10 @@ type applicationService struct {
 func NewApplicationService(client kubernetes.Interface, namespace string, config loader.Config, dnsService *dns.Service, computeService *compute_v1.Service, serviceManagement *servicemanagement.APIService) ApplicationServiceInterface {
 	a := new(applicationService)
 	a.clientSet = client
+	a.prefixedNamespace = namespace
```
Comment (jazzzzy) [inline service/app.go:48]: why don't we overwrite the a.namespace?

**Review by jazzzzy — COMMENTED** [review]
@Engerim I added just 2 minor comments. Please take a look.

**Review by jazzzzy — APPROVED** [review]

**Pull request merged**